### PR TITLE
Disable grace period popup during UI tests

### PR DESCRIPTION
**Pull request by phil-davis**
Let's see if this helps.

### CODE REVIEW

**Comment by phil-davis** [comment]
Not in the right place - it needs to be done from the directory of the server-under-test (not from the directory where the test-runner code lives)

File: .drone.star [hunk 1]
```diff
@@ -755,6 +755,8 @@ def ui(config):
     },
     'commands': [
       'curl -k -s -u admin:admin http://server:8080/ocs/v2.php/apps/testing/api/v1/occ -d "command=config:system:set --type boolean --value false grace_period.demo_key.show_popup"',
+      'curl -k -s -u admin:admin http://server:8080/ocs/v2.php/apps/testing/api/v1/occ -d "command=app:disable ownbrander"',
```
Comment (jnweiger) [inline .drone.star:758]: close! its not ownbrander. it's theme-enterprise that deserves a disable :-)

**Comment by phil-davis** [comment]
The finally-needed magic has been put into #190

**Comment by jnweiger** [comment]
... and has also been put into https://github.com/owncloud-docker/server/pull/192

**Pull request closed without merging**